.gitlab-ci.yml

```diff
@@ -42,10 +42,11 @@ publish:versions:
     - hub clone mendersoftware/mender-docs-site && mv versions.json mender-docs-site/versions.json && cd mender-docs-site
     - git checkout -b update-versions-$(date +%s)
     - "git add versions.json && git commit --signoff -m 'chore: Version information update'"
-    - hub pull-request --push --draft --base mendersoftware:master --message "Version information update" --message "keeping up with the versions"
+    - hub pull-request --push --base mendersoftware:master --message "Version information update" --message "keeping up with the versions"
   artifacts:
     expire_in: 2w
     paths:
       - versions.json
+    when: always
```

Comment:
oh yeah, I honestly always need the always, it should be the default. always. ;-)